README.md

```diff
@@ -0,0 +1,144 @@
+named environment (like `default` or `production`) and a set of AWS resources
+associated with that environment. For example:
+
+```python
+ECSMANAGE_ENVIRONMENTS = {
+    'default': {
+        'TASK_DEFINITION_NAME': 'StagingAppCLI',
+        'CLUSTER_NAME': 'ecsStagingCluster',
+        'LAUNCH_TYPE': 'FARGATE',
+        'SECURITY_GROUP_TAGS': {
+            'Name': 'sgAppEcsService',
+            'Environment': 'Staging',
+            'Project': 'ProjectName'
+        },
+        'SUBNET_TAGS': {
+            'Name': 'PrivateSubnet',
+            'Environment': 'Staging',
+            'Project': 'ProjectName'
+        },
+        'AWS_REGION': 'us-east-1',
+    },
+}
+```
+
+This configuration defines a single environment, named `default`, with
+associated AWS ECS resources.
+
+### Environments
+
+The key name for an environment can be any string. You can use this name
+with the `--env` flag when running the command to run a command on a
+different environment. Take this `ECSMANAGE_ENVIRONMENTS` configuration
+as an example:
+
+```python
+ECSMANAGE_ENVIRONMENTS = {
+    'default': {
+        'TASK_DEFINITION_NAME': 'StagingAppCLI',
+        'CLUSTER_NAME': 'ecsStagingCluster',
+        'SECURITY_GROUP_TAGS': {
+            'Name': 'sgStagingAppEcsService',
+        },
+        'SUBNET_TAGS': {
+            'Name': 'StagingPrivateSubnet',
+        },
+    },
+    'production': {
+        'TASK_DEFINITION_NAME': 'ProductionAppCLI',
+        'CLUSTER_NAME': 'ecsProductionCluster',
+        'SECURITY_GROUP_TAGS': {
+            'Name': 'sgProductionAppEcsService',
+        },
+        'SUBNET_TAGS': {
+            'Name': 'ProductionPrivateSubnet',
+        },
+    },
+}
+```
+
+This configuration defines two environments, `default` and `production`.
+Using the above settings, you could run production migrations with the
+following command:
+
+```
+$ django-admin ecsmanage --env production migrate
+```
+
+If the `--env` argument is not present, the command will default to the
+environment named `default`. 
+
+### AWS Resources
+
+The following environment configuration keys help the management command locate
+the appropriate AWS resources for your cluster:
+
+| key name | description | default |
+| -------- |  ----------- | ------- |
+| `TASK_DEFINITION_NAME` | The name of your ECS task definition. The command will automatically retrieve the latest definition. | | 
+| `CLUSTER_NAME` | The name of your ECS cluster. | | 
+| `SECURITY_GROUP_TAGS` | A dictionary of tags to use to identify a security group for your task. | |
+| `SUBNET_TAGS` | A dictionary of tags to use to identify a subnet for your task. | |
+| `LAUNCH_TYPE` | The ECS launch type for your task. | `FARGATE` |
+| `AWS_REGION` | The AWS region to run your task. | `us-east-1` |
+
+## Developing
+
+Local development is managed with Python virtual environments. Make sure that
+you have [Python 3.6+ and pip installed](https://www.python.org/downloads/)
```

Comment:
Because of the use of f-strings in the module, support is limited to Python 3.6+. I couldn't decide how much of a downside this presented.

If we decide to support current Python 3 versions (3.4+), we'll need to refactor `ecsmanage/management/commands/ecsmanage.py` to remove the use of f-strings. If we decide to support Python 2.7 as well (which will be retired next January), we'll also need to refactor the development environment to not use the standard library virtualenv implementation, which was introduced in Python 3.4.